openshift/aws-account-operator
# [OSD-17631] - boilerplate update

# What is being added?
_Is this a fix for a bug?  What's the bug?  Is this a new feature? Please describe it. Is this just a small typo fix?  That's fine too!_
This is a boilerplate update

## Checklist before requesting review

- [ N/A ] I have included unit tests
- [ N/A ] I have updated any corresponding documentation

## Steps To Manually Test
Operator should run on a fips enabled cluster

Ref [OSD-17631](https://issues.redhat.com//browse/OSD-17631)
